src/System.Data.SqlClient/tests/Tools/TDS/TDS.Clients.Console/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("TDS.Clients.Console")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyProduct("TDS.Clients.Console")]
+[assembly: AssemblyCopyright("Copyright © Microsoft 2011")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
```

Comment:
I will remove the TDS.Clients.Console and TDS.Servers.Console from this PR. They are not needed immediately for the TDS server to operate. 
I will get them in later if needed for any test use cases. Thanks for catching this.